### PR TITLE
Migrate presubmits of the release repo

### PR DESCRIPTION
**Pull request by hongkailiu**
[This query](https://prometheus-prow-monitoring.apps.ci.l2s4.p1.openshiftapps.com/graph?g0.range_input=1h&g0.expr=sum(increase(prowjob_state_transitions%7Bstate%3D%22pending%22%2Ctype%3D%22presubmit%22%7D%5B7d%5D))%20by%20(org%2C%20repo)&g0.tab=1) is used for getting the data.

Those are monsters:

```
{org="openshift",repo="release"} | 22027.25874
{org="cri-o",repo="cri-o"} | 2576.323183
{org="openshift",repo="installer"} | 1379.876803
{org="openshift",repo="console"} | 1373.43433
{org="openshift",repo="origin"} | 1067.075557
{org="openshift",repo="machine-config-operator"} | 891.9290575
{org="openshift",repo="cluster-network-operator"} | 764.9462903
{org="openshift",repo="ovn-kubernetes"} | 598.8134571
{org="openshift",repo="oc"} | 504.7741215
{org="openshift",repo="ci-tools"} | 461.4647364
{org="openshift",repo="ocs-operator"} | 408.4426416
{org="openshift",repo="router"} | 408.2264081
```

Maybe start with our own repo this time.

/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

**Comment by hongkailiu** [comment]
/hold

Need @stevekuznetsov to clarify the date of freeze before merge.

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**
